apps/rush-lib/src/scripts/install-run-rushx.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See the @microsoft/rush package's LICENSE file for license information.
+
+// THIS FILE WAS GENERATED BY A TOOL. ANY MANUAL MODIFICATIONS WILL GET OVERWRITTEN WHENEVER RUSH IS UPGRADED.
+//
+// This script is intended for usage in an automated build environment where the Rush command may not have
+// been preinstalled, or may have an unpredictable version.  This script will automatically install the version of Rush
+// specified in the rush.json configuration file (if not already installed), and then pass a command-line to it.
+// An example usage would be:
+//
+//    node common/scripts/install-run-rushx.js install
+//
+// For more information, see: https://rushjs.io/pages/maintainer/setup_new_repo/
+
+import * as path from 'path';
+import * as fs from 'fs';
+
+import {
+  installAndRun,
+  findRushJsonFolder,
+  RUSH_JSON_FILENAME,
+  runWithErrorAndStatusCode
+} from './install-run';
+
+const PACKAGE_NAME: string = '@microsoft/rush';
+
+function getRushVersion(): string {
```

Comment:
Almost all of this logic is identical to `install-run-rush`. We should just have this call an exported function from that script, but pass in the `rushx` bin instead of `rush`.

Reply:
done